performance_engine/config/config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"fields" : {
+		"ClientInc" : "Portfolio/JLH/ClientInc",
```

Comment:
is `JLH` intrinsically tied into the code?

Reply:
Nope.